### DISCUSSION TITLE
Msprime/SLiM/Msprime Conversion with admixed populations

### DISCUSSION THREAD

**Discussion by albrenner** [tskit-dev/pyslim#334] [Q&A]
Hello!
I am trying to simulate admixture in a population with an extensive demographic history in Msprime, then upload it in SLiM, run simulations for 8 generations, and then move the tree sequence back to Msprime for neutral mutation overlay. I have done all of these steps successfully however once I try and make vcf files out of the individuals at the end of all of these steps I get the error "Individual has nodes that are sample and non-samples".
After consulting with people who know way more about coding in these programs than me, they see no issue with the codes I have done and worry about it being a pyslim conversation issue. Does anyone have experience doing something similar to this?
Thank you!
-Alaina Brenner, PhD student at UC Davis

**Comment by bhaller** [comment]
Hi Alaina!  I don't know the answer, but the first steps to debug this would probably be (1) trying to construct a minimal reproducible example by simplifying your model/code down as much as possible (while still reproducing the bug), and then (2) posting the code for that minimal reproducible example here.  That will allow @petrelharp to run your model on his side to look into the problem.  I'd also recommend making sure that your SLiM, tskit, msprime, and pyslim are all current, to eliminate the possibility of a problem due to a version mismatch.  Good luck!

**Comment by petrelharp** [comment]
I agree. And: in what you're describing there's lots of opportunities for indexing errors that are very hard to spot - most likely, the individuals you're asking for a VCF of are not in fact alive at the end of the simulation (or otherwise Remembered). To identify this sort of thing, do lots of interactive printing out of things.
